model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/target_platform_capabilities.py

```diff
@@ -38,13 +39,15 @@ class TargetPlatformCapabilities(ImmutableClass):
     def __init__(self,
                  tp_model: TargetPlatformModel,
                  name: str = "base",
-                 version: str = None):
+                 version: str = None,
+                 add_metadata: bool = False):
```

Comment:
Didn't we say that it should be in the tp model? is there a reason to put it in the tpc?